macros/models/_sector/dex/uniswap_compatible_pools.sql

```diff
@@ -0,0 +1,43 @@
+{% macro uniswap_compatible_pools(
+    blockchain = null
+    , project = null
+    , version = null
+    , hardcoded_fee = null 
+    , fee_column_name = null
+    , Factory_evt_PairCreated = null
+    , pool_column_name = 'pair'
+    , token0_column_name = 'token0'
+    , token1_column_name = 'token1'
+    )
+%}
+
+SELECT 
+    '{{ blockchain }}' AS blockchain
+    , '{{ project }}' AS project
+    , '{{ version }}' AS version
+    , f.{{ pool_column_name }} as pool
+    {% if hardcoded_fee %} -- use hardcoded fee if it's exists
+    , {{ hardcoded_fee }} as fee
+    {% endif %}
+    {% if fee_column_name %}
+    , f.{{fee_column_name}} as fee -- use fee column if hardcoded fee doesn't exists
+    {% endif %}
+    {% if not (fee_column_name or hardcoded_fee) %}
+    , CAST(NULL as BIGINT) as fee 
+    {% endif %}
+    , array_agg(
```

Comment:
Is it possible to extend this `tokens` column into `token_symbols`?

Reply:
hmm sure @Hosuke, I will add it in

Reply:
token_symbols added @Hosuke